plugin/trino-hive/src/main/java/io/trino/plugin/hive/coercions/TimestampCoercer.java

```diff
@@ -35,6 +37,7 @@
 import static java.time.ZoneOffset.UTC;
 import static java.time.format.DateTimeFormatter.ISO_LOCAL_DATE;
 import static java.time.format.DateTimeFormatter.ISO_LOCAL_TIME;
+import static org.joda.time.DateTimeConstants.SECONDS_PER_DAY;
```

Comment:
Why not `io.trino.spi.type.Timestamps#SECONDS_PER_DAY` ?

Reply:
Thanks for pointing it out - Have updated it.